README.md

```diff
@@ -63,6 +63,45 @@ Requests to cross-scheme subresources will no longer send `SameSite=Lax` or `Sam
 For example, if http://site.example embeds an image from https://site.example, `SameSite=Lax` and `SameSite=Strict` cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
 ## Questions
+### How does this affect iframes?
+#### The src attribute
+A request to get the content specified by the `src` attribute is treated as any other subresource request and is affected the same ways.
+Once the content is loaded it runs as cross-site or same-site depending on if the `src` value was cross-scheme or not, respectively.
+
+For example, the following request for a page will not be sent `SameSite=Lax` nor `SameSite=Strict` cookies (nor will its response be able to set those cookies) as it's cross-scheme. After the page is loaded nothing on the page is able to access those `SameSite` cookies because it's being run as cross-site.
+```
+<html>
+<body>
+
+<h1>This page is hosted at https://website.example/secure.html</h1>
+<iframe src="http://website.example/insecure.html" title="An iframe to an insecure page"></iframe>
+
+</body>
+</html>
+```
+#### The srcodc attribute
```

Comment:
srcdoc

Reply:
Thanks for catching this. Even with your comment I still missed it during a few proof reads...